text/mavlink2_1.md

```diff
@@ -0,0 +1,62 @@
+  * Start date: 2022-10-12
+  * Contributors: Andrew Tridgell
+
+# Summary
+
+Rough proposal for MAVLink 2.1
+
+# Motivation
+
+Key features wanted:
+
+ * support for more than 256 system IDs
```

Comment:
The extra complexity about this doesn't really seem worth it to me. Is there a larger problem with just being looser about sys/comp id strictness, and at an application level just treating them as a single touple? (IE we already have 65535 possible unique addresses).

I'm guessing this may break something in software I haven't thought of from the top of my head, but on first glance I'd prefer to not add the extra responsibility to everything.